config/action.d/firewallcmd-multiport.conf

```diff
@@ -3,17 +3,18 @@
 # Author: Donald Yandt 
 # Because of the --remove-rules in stop this action requires firewalld-0.3.8+
 
+
 [INCLUDES]
 
-before = iptables-blocktype.conf
+before = iptables-common.conf
 
 [Definition]
 
 actionstart = firewall-cmd --direct --add-chain ipv4 filter f2b-<name>
               firewall-cmd --direct --add-rule ipv4 filter f2b-<name> 1000 -j RETURN
-              firewall-cmd --direct --add-rule ipv4 filter <chain> 0 -m state --state NEW -p <protocol> -m multiport --dports <port> -j f2b-<name>
+              firewall-cmd --direct --add-rule ipv4 filter <chain> 0 -m conntrack --ctstate NEW -p <protocol> -m multiport --dports <port> -j f2b-<name>
```

Comment:
I could be wrong but this is what I've gathered on nf_conntrack support:

- in modules.py function get_firewall_modules() it just reads from "/proc/modules" and loads dependencies if they exist so no compatibility checks.

- further research into all the talk about firewalld getting conntrack support suggests that they are talking about nf_conntrack userspace tools and not the kernel conntrack module.

- all the services have nf_conntrack module names. Example "<module name="nf_conntrack_amanda"/>"

with that stated and module "state"  being depreciated I suggest we move to nf_conntrack.